packages/data-model/src/types.ts

```diff
@@ -20,14 +20,14 @@
 // Constants
 
 export enum ApplicationStates {
-	'DRAFT',
-	'INSTITUTIONAL_REP_REVIEW',
-	'DAC_REVIEW',
-	'DAC_REVISIONS_REQUESTED',
-	'REJECTED',
-	'APPROVED',
-	'CLOSED',
-	'REVOKED',
+	DRAFT = 'DRAFT',
+	INSTITUTIONAL_REP_REVIEW = 'INSTITUTIONAL_REP_REVIEW',
+	DAC_REVIEW = 'DAC_REVIEW',
+	DAC_REVISIONS_REQUESTED = 'DAC_REVISIONS_REQUESTED',
+	REJECTED = 'REJECTED',
+	APPROVED = 'APPROVED',
+	CLOSED = 'CLOSED',
+	REVOKED = 'REVOKED',
```

Comment:
Important to use these named enum values if we are using TS enums, so thank you for this update. We can take a moment to review our enum usage in a future change.